src/tests/fuzz/fuzz_applayerparserparse.c

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file
+ * @author Philippe Antoine <contact@catenacyber.fr>
+ * fuzz harness for AppLayerProtoDetectGetProto
+ */
+
+
+#include "suricata-common.h"
+#include "app-layer-detect-proto.h"
+#include "flow-util.h"
+#include "app-layer-parser.h"
+
+#define HEADER_LEN 6
+
+void fuzz_openFile(const char * name) {
```

Comment:
this function is not necessary.

Reply:
Indeed, it is not used yet.
In other projects, I use this function to open a file for logging.

How do you achieve this ?